vehicle/raw_vehicle_cmd_converter/src/node.cpp

```diff
@@ -204,20 +200,30 @@ double RawVehicleCommandConverterNode::calculateBrakeMap(
   return desired_brake_cmd;
 }
 
-void RawVehicleCommandConverterNode::onSteering(const Steering::ConstSharedPtr msg)
+void RawVehicleCommandConverterNode::processSteering(const Steering::ConstSharedPtr msg)
 {
+  if (!msg) {
+    return;
+  }
   current_steer_ptr_ = std::make_unique<double>(msg->steering_tire_angle);
 }
 
-void RawVehicleCommandConverterNode::onVelocity(const Odometry::ConstSharedPtr msg)
+void RawVehicleCommandConverterNode::processOdometry(const Odometry::ConstSharedPtr msg)
 {
+  if (!msg) {
+    return;
+  }
   current_twist_ptr_ = std::make_unique<TwistStamped>();
   current_twist_ptr_->header = msg->header;
   current_twist_ptr_->twist = msg->twist.twist;
 }
```

Comment:
IMO
They are no longer a callback, but simply an assignment operation, I prefer to sum up in `onControlCmd`.
Actually, `onControlCmd` can be replaced to timer callback, but this time I believe that is out of this PR and look forward you to the future contribution.

Reply:
Thank you for your review. I sum up the functions.